mvn/archetypes/radix/src/main/resources/archetype-resources/workflow/src/main/resources/etc/logging.properties

```diff
@@ -16,53 +16,54 @@
 # Specify the handlers to create in the root logger
 # (all loggers are children of the root logger)
 # The following creates two handlers
-handlers = java.util.logging.ConsoleHandler, java.util.logging.FileHandler
+handlers=java.util.logging.ConsoleHandler, java.util.logging.FileHandler
```

Comment:
Same as earlier. Were these changes necessary?

Reply:
no, it was not. these were made by the IDE which I had mistakenly staged for commit. resolved. 👍